src/CSET/operators/plot.py

```diff
@@ -320,6 +320,43 @@ def _plot_and_save_line_series(
     plt.close(fig)
 
 
+def _plot_and_save_vertical_line_series(
+    cube: iris.cube.Cube, coord: iris.coords.Coord, filename: str, title: str, **kwargs
+):
+    """Plot and save a 1D line series.
+
+    Parameters
+    ----------
+    cube: Cube
+        1 dimensional Cube of the data to plot on x-axis.
+    coord: Coord
+        Coordinate to plot on y-axis.
+    filename: str
+        Filename of the plot to write.
+    title: str
+        Plot title.
+    """
+    fig = plt.figure(figsize=(8, 8), facecolor="w", edgecolor="k")
+    iplt.plot(cube, coord, "o-")
+    plt.gca().invert_yaxis()
+    ax = plt.gca()
```

Comment:
```suggestion
    ax = plt.gca()
    ax.invert_yaxis()
```
We've already grabbed the axes, so let's use it.